### PR TITLE
Add version compatibility for 1.5

### PR DESCRIPTION
**Pull request by SIkebe**
Sorry for inconvenience. Should have done in #27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ See [Maven Repository](http://central.maven.org/maven2/org/codelibs/gitbucket/gi
```
Comment (kw-udon) [inline README.md]: Would you mind updating the link to the Maven Repository as well?
`gitbucket-fess-plugin_2.12` -> `gitbucket-fess-plugin_2.13`

**Comment by kw-udon** [comment]
Thanks for this change! It's good timing as we usually update README after releasing:)

File: README.md (hunk 1 above)
Comment (SIkebe) [reply README.md]: done!

**Comment by kw-udon** [comment]
Thanks!

**Pull request merged**